### PR TITLE
Added ODF ReadStep extension for Parquet format

### PR DESCRIPTION
**Pull request by zaychenko-sergei**

### CODE REVIEW

File: opendatafabric/src/serde/flatbuffers/proxies_generated.rs [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 // automatically generated by the FlatBuffers compiler, do not modify
 
-use std::cmp::Ordering;
-use std::mem;
+use core::cmp::Ordering;
```
Comment (sergiimk) [inline opendatafabric/src/serde/flatbuffers/proxies_generated.rs:12]: Can you check which version of Flatbuffers compiler you are using? I we use different ones we'll be producing too much diffs.

**Review by sergiimk — COMMENTED** [review]
Looks as expected so far, but we'll need to bump the Spark engine image version once we release it.

File: opendatafabric/src/serde/flatbuffers/proxies_generated.rs (hunk 1 above)
Comment (sergiimk) [reply opendatafabric/src/serde/flatbuffers/proxies_generated.rs:12]: Mine is:
```bash
$ flatc --version
flatc version 2.0.0

$ yay -Q | grep flat
flatbuffers-git 2358-1
```
I moved to latest git version after encountering some issues in Rust codegen, but can switch back to a latest stable if it is indeed stable :)

Comment (zaychenko-sergei) [reply opendatafabric/src/serde/flatbuffers/proxies_generated.rs:12]: I'm using the latest version available on today, 2.0.6

File: kamu-core/tests/tests/engine/test_ingest_engine.rs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+// Copyright Kamu Data, Inc. &&and contributors. All rights reserved.
```
Comment (sergiimk) [inline kamu-core/tests/tests/engine/test_ingest_engine.rs:1]: accidental?

**Review by sergiimk — APPROVED** [review]
One small thing, otherwise LGTM!

**Pull request merged**